server/routes/api/choropleth.py

```diff
@@ -132,43 +148,40 @@ def choropleth_api():
                                     geojson['coordinates'],
                                     geojson['type']))
             # Process Statistical Observation if valid.
-            if ('data' in stat_var_by_geo.get(geo_id, []) and
-                    (not per_capita or
-                    'data' in population_by_geo.get(geo_id, []))):
+            if ('data' in population_by_geo.get(geo_id, [])):
                 # Grab the latest available data.
-                stat_obs = next(iter(
-                    reversed(stat_var_by_geo[geo_id]['data'].values())))
-
-                # Handle per-capita option.
-                if per_capita:
-                    # Note that this could be from a different period from the
-                    # statistical variable.
-                    # TODO(iancostello): Determine how to handle populations 
-                    # and statistical values from different times.
-                    pop = next(iter(
-                        reversed(population_by_geo[geo_id]['data'].values())))
-                    stat_obs /= pop
-
-                values.append(stat_obs)
-                geo_feature["properties"]["value"] = stat_obs
+                pop = next(iter(
+                    reversed(population_by_geo[geo_id]['data'].values())))
+                geo_feature["properties"]["pop"] = pop
```

Comment:
```suggestion
                geo_feature["properties"]["pop"] = next(iter(
                    reversed(population_by_geo[geo_id]['data'].values())))
```

Typically, one avoids defining a one-use variable.

Reply:
Removed. It was more of a relic of the old code